tests/unit/test_loadbalance.py

```diff
@@ -6,8 +6,6 @@
 from jina.enums import SchedulerType
 from tests import random_docs
 
-os.environ['JINA_LOG_LEVEL'] = 'DEBUG'
```

Comment:
you would need to change this back to info with a fixture.

However, I would like to have DEBUG logs when testing for the tests? what do u think? we could have a general fixture for that?

Reply:
agreed, done! @JoanFM